api/nodes/views.py

```diff
@@ -1282,6 +1282,8 @@ class NodeFilesList(generics.ListAPIView, WaterButlerMixin, ListFilterMixin, Nod
         base_permissions.TokenHasScope,
     )
 
+    ordering = ('materialized_path',)  # default ordering
```

Comment:
Alternatively, I could add a `date_modified` to each class with it's specific implementation. That seems better to me at second thought.